Campaign/management/commands/StartNewCampaign.py

```diff
@@ -145,6 +145,7 @@ def handle(self, *args, **options):
 
         batches_json = options['batches_json']
         campaign_name = context['CAMPAIGN_NAME']
+        campaign_opts = context.get('TASK_OPTIONS', '')

```

Comment:
Rename to `task_options`?

Reply:
Or change environment variable name?

Reply:
I was thinking about that. I think that TASK_OPTIONS is more descriptive for `manifest.json` as the modifications are applied at task level anyway. However, I added a new field `campaignOptions` to the `Campaign` object and that's mainly because there seems to be no good parent class for all task objects - so in this line `campaign_opts` may be more appropriate. I personally prefer cleaner API than backend :) But will happily change if you think otherwise.